javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,11 @@
+// ******* Ez Namey *******
+
+const firstWords = ["Easy", "Awsome", "Evolutionary", "Approachable", "Creative", "Authentic", "Trustworthy", "Motivated", "Professional", "Reliable"];
+const secondWords = ["Company", "Innovation", "Factory", "Corporation", "World", "House", "Future", "Future Planners", "Dream", "Business"];
+
+
+const randomFirstWordIndex = Math.floor(Math.random() * 10) + 0;
```

Comment:
```suggestion
const randomFirstWordIndex = Math.floor(Math.random() * firstWords.length) ;
```
It's better to avoid using constants in cases like this when you have to run some code that relies on an array that can change it's size

Reply:
Thanks. I just used the code snippet provided in the exercise description. It was already too advanced for me. I'll fix and push.

Reply:
Yes. I saw that. Reviews are not ment for pointing problems. Are used to show also how can you do thinks differently and also learn from more experience developers

Reply:
Perfect! Thanks.